pkg/fileservice/s3_fs.go

```diff
@@ -328,8 +329,7 @@ func (s *S3FS) Read(ctx context.Context, vector *IOVector) (err error) {
 	default:
 	}
 
-	ctx, span := trace.Start(ctx, "S3FS.Read")
-	defer span.End()
+	bmetrics.Get().Incr("S3FsRead")
```

Comment:
Why this is hot simply SomeCounter.Incr()?

Reply:
Simple Metrics Counter increment is implemented here. https://github.com/matrixorigin/matrixone/pull/8253/files#r1125603080